app/src/main/res/values/strings.xml

```diff
@@ -793,6 +793,8 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_air_pump_bicycle_repair_station_title">Is there a working bicycle pump here?</string>
     <string name="quest_air_pump_compressor_title">Is there an air compressor available here?</string>
 
+    <string name="quest_board_name_title">"What’s the title of this information board?"</string>
```

Comment:
There are no quotation marks anywhere else in the string, do these really need to be here?

Reply:
I have no idea whether `’` counts, and I had problems with some strings without quotes in the past, so I'm playing it safe since... 

And, it was copied from `<string name="quest_busStopName_title2">"What’s the name of this stop?"</string>` which was pretty similar and had quotes.

Anyway cleanup was suggested before in https://github.com/streetcomplete/StreetComplete/issues/4334, but was deemed _"very very very unimportant"_ so I won't worry too much about that one. Thanks for review though!